pkgs/development/python-modules/pur/default.nix

```diff
@@ -1,19 +1,26 @@
 { lib
-, buildPythonPackage
-, click
+, python3
 , fetchFromGitHub
-, pytestCheckHook
 }:
 
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      click = self.callPackage ../../../development/python-modules/click/7.nix { };
+    };
+  };
+in
+with py.pkgs;
```

Comment:
can this use of with be replaced by a let?
https://nix.dev/anti-patterns/language#with-attrset-expression

Reply:
Is it better now?